packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -125,12 +125,9 @@ const DefaultControls = ( props ) => {
 const MenuInspectorControls = ( props ) => {
 	const isOffCanvasNavigationEditorEnabled =
 		window?.__experimentalEnableOffCanvasNavigationEditor === true;
-	const menuControlsSlot = window?.__experimentalEnableBlockInspectorTabs
-		? 'list'
-		: undefined;
 
 	return (
-		<InspectorControls __experimentalGroup={ menuControlsSlot }>
+		<InspectorControls __experimentalGroup="list">
```

Comment:
With the off canvas experiment turned off it results in a List View tab that doesn't have a List View within it:
![Screen Shot 2023-01-11 at 11 13 10 am](https://user-images.githubusercontent.com/677833/211709557-5cbfec4e-2776-44dd-b756-38b9ed7dc7dc.png)

So I wonder if the addition of the List View tab should now be dependent on the off canvas experiment being enabled.

Other than that, this PR looks good 👍

Reply:
Good catch, thank you!

I'll fix it up along with the failing e2es.

Reply:
Looks like I'll also need to revert the change around checking for list view tab fills so we don't get an empty tab.

Reply:
✅  Done in https://github.com/WordPress/gutenberg/pull/47045/commits/f8d7d08a88ae10119600b7b178179bd7c946832f